### PR TITLE
Update Java example

### PR DESCRIPTION
**Pull request by jessdelacruzsantos**
Switches out old SDK for the new SDK in existing connect examples

### CODE REVIEW

File: connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java [hunk 1]
```diff
@@ -78,35 +74,41 @@ public static void main(String[] args) throws Exception {
...
 
         return value;
     }
 
     @RequestMapping("/")
     String index(Map<String, Object> model) throws ApiException {
-        model.put("paymentFormUrl", squareEnvironment.equals("sandbox") ?
-            "https://js.squareupsandbox.com/v2/paymentform" : "https://js.squareup.com/v2/paymentform");
+        model.put("paymentFormUrl",
+                squareEnvironment.equals("sandbox") ? "https://js.squareupsandbox.com/v2/paymentform"
```
Comment (hukid) [inline connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java]: is  `squareClient` tell the environment information? Wondering if comparing strings can be replaced by environment check?

File: connect-examples/v2/java_payment/src/main/resources/templates/charge.html [hunk 2]
```diff
@@ -9,9 +9,12 @@
 </head>
 
 <body>
-  <p>
-    <span th:text="${payment}"></span>
-  </p>
+  <h1> Charge Result</h1>
```
Comment (hukid) [inline connect-examples/v2/java_payment/src/main/resources/templates/charge.html]: Did this new SDK change the error behavior?

File: connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java (hunk 1 above)
Comment (jessdelacruzsantos) [reply connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java]: I'm not too sure what your question is here. The string comparison is of an environment variable.

Comment (hukid) [reply connect-examples/v2/java_payment/src/main/java/com/squareup/connectexamples/ecommerce/Main.java]: Never mind, this is fine.

File: connect-examples/v2/java_payment/src/main/resources/templates/charge.html (hunk 2 above)
Comment (jessdelacruzsantos) [reply connect-examples/v2/java_payment/src/main/resources/templates/charge.html]: I went ahead and added error handling for ApiExceptions and created an error page to better display the error to the user.

Comment (hukid) [reply connect-examples/v2/java_payment/src/main/resources/templates/charge.html]: Nice improvement!

**Pull request merged**